Replace AppImage extraction instructions with tar.xz archives (#444)
sinelaw/fresh: README.md

Download the latest release for your platform from the [releases page](https://github.com/sinelaw/fresh/releases).

#### Linux with GLibC

> [!TIP]
> Replace `$ARCH` below with either `x86_64` or `aarch64` depending on your CPU architecture.
>

> > And replace `$VERSION` below with the desired version (usually the latest one).

Author: ```suggestion
> And replace `$RELEASE_TAG` below with the desired release tag (usually the latest one).
```

> **Step 0:** Determine the architecture and release tag to download (if you are too lazy to fulfill the tip above).

Author: ```suggestion
**Step 0:** Determine the architecture and release tag to download if you are too lazy to fulfill the tip above (`jq` required).
```

```bash

> ARCH=$(uname -m)

Author: ```suggestion
ARCH=$(uname -m | sed -e 's/arm64/aarch64/' -e 's/armv8l/aarch64/)
```

> RELEASE_TAG=$(curl -s https://api.github.com/repos/sinelaw/fresh/releases/latest | grep -Po '"tag_name": "\K[^"]*')

Author: ```suggestion
RELEASE_TAG=$(curl -s https://api.github.com/repos/sinelaw/fresh/releases/latest | jq -r .tag_name)
```

> RELEASE_TAG=$(curl -s https://api.github.com/repos/sinelaw/fresh/releases/latest | jq -r .tag_name)

Author: There is a "more portable" command with `grep` and `sed` instead of `jq` but I don't like it because it looks too error-prone.
